Jenkinsfile

```diff
@@ -21,13 +21,14 @@ def releaseTag = new com.socrata.ReleaseTag(steps, service_server)
 pipeline {
   options {
     ansiColor('xterm')
+    disableConcurrentBuilds(abortPrevious: true)
   }
   parameters {
+    string(name: 'AGENT', defaultValue: 'build-worker', description: 'Which build agent to use?')
```

Comment:
We typically list the agent and branch specifier first.

Reply:
Very minor: The description doesn't seem like a question.